delta-lake-walkthrough/delta-lake-walkthrough.py

```diff
@@ -224,12 +218,13 @@ def cleanup_paths_and_tables():
 
 # COMMAND ----------
 
-# MAGIC %md
-# MAGIC Use the above output to create a table from the parquet location.
+# MAGIC %md Use the above output to create a table from the parquet location.
 
 # COMMAND ----------
 
-# MAGIC %sql CONVERT TO DELTA parquet.`/FileStore/YOURUSERNAME/deltaDemo/loans_parquet/`
+# COMMAND ----------
+
+# MAGIC %sql CONVERT TO DELTA parquet.`/FileStore/syedalimasroor.r/deltaDemo/loans_parquet/`
```

Comment:
Can you change this back to `YOURUSERNAME` please?

Reply:
Changes are done now and commited. Can you please check now and merge the PR ?